### PR TITLE
Episode 1 transcript

### PR DESCRIPTION
**Pull request by ole**
Following up on #15, this is a machine-generated transcript for episode 1 in [WebVTT format](https://en.wikipedia.org/wiki/WebVTT).

I have already edited the first ~10 minutes of the transcript. The rest is still full of mistakes (but perfectly readable). It would be great to see other contributors chip in and edits a portion (the work is not so bad if you do it in 5-minute portions).

**If you want to help with editing, please add a comment to this PR with the segment you'd like to take on _before_ you start. We want to avoid duplication of work. ~You should be able to push your changes directly to this PR.~**

_Edit: my instructions were wrong. I forgot that only people with write access to this repo or my fork can push to this branch. I'll invite people who comment on this PR to my fork so you can push your edits to the `ole:episode-1-transcript` branch. Then they should appear here._

Many thanks to @JulianKahnert for working with me [on the tool for generating the WebVTT text](https://github.com/ole/transcribe).

Let me know if you want to rename the file or move it to another directory. I haven't added a link to the transcript to the shownotes yet. Let's first figure out where the transcript files should be located and how we want to name them.

### CODE REVIEW

**Comment by ole** [comment]
I just [sent a tweet asking people for help with the editing](
https://twitter.com/olebegemann/status/1094988805608755203). It would be good if we could keep this PR open for a while so people can push their edits directly to it. Thanks!

**Comment by vincentthe0ne** [comment]
I would like to help with the editing 💪🏻. I would start from 09:46.529 to ~15:00.000 (+/- depending on the last Speaker)

**Comment by MonicaG** [comment]
Hello! I would like to help editing.  I would like to do 00:59:15.213 -> 01:07:26.240.

**Comment by ole** [comment]
@vincentthe0ne Sorry, my original instructions ("push directly to this PR") were wrong. I just invited you as a collaborator to my fork of the repo. You should be able to push your edits to the `episode-1-transcript` branch there and then they should appear here.

**Comment by MonicaG** [comment]
I'll do 00:39:55.214 ->  00:50:18.494

**Comment by ole** [comment]
I'll pick up at 00:15:14.437 until 00:23:31.535.

**Comment by ole** [comment]
I'll do the next batch from 00:23:32.785 to 00:30:32.178.

**Comment by ole** [comment]
I'll now do the remaining bit from 00:30:32.658 to 00:39:54.794.

**Comment by ole** [comment]
Editing is done.

@garricn @lattner @JohnSundell This is ready for merging if you ask me.

**Comment by ole** [comment]
Yay! Thanks again for your help @vincentthe0ne and especially @MonicaG!

**Comment by lattner** [comment]
Amazing, this is really fantastic work, thank you so much for driving this forward!

**Comment by BasThomas** [comment]
Awesome stuff! Congrats @ole, @vincentthe0ne & @MonicaG!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: Shownotes/Episode1-Transcript.vtt [hunk 1]
```diff
@@ -0,0 +1,710 @@
...
+
+00:30:32.658 --> 00:31:09.181
+<v Chris Lattner>That's a perfect segue into the last group I want to talk about. Because, you look at what Swift is and sure, it's a compiler, it's a language spec, it's a set of APIs, it's and IDE. But the thing that makes it real and the thing that reached so many people is the work done by the Developer Publication Group. These are the tech writers at Apple that wrote things such as the Swift Programming Language book. And the success and the rapid adoption of Swift is in huge part directly related to
+
+00:31:10.071 --> 00:31:18.041
+<v Chris Lattner>such high quality good documentation and a book being available on day one. And it's continued to be maintained today, and it's amazing.
+
+00:31:18.571 --> 00:31:34.451
+<v Chris Lattner>And part of that was that we pulled in the tech writers right into the design meetings. Folks like Tim Isted and Brian Lanyon[?], Alex Martini spent a tremendous amount of time in weekly meetings where we were arguing about, "do we use
```
Comment (ole) [inline Shownotes/Episode1-Transcript.vtt]: @lattner I'm not quite sure I got this name right: "Brian Lanyon". Can you confirm?

**Pull request merged**